android/app/src/main/res/values/strings.xml

```diff
@@ -136,10 +136,6 @@
     <string name="five_plus">5+</string>
     <string name="roommate_preference">Roommate Preference</string>
     <string name="all_filters">All Filters</string>
-
-
-
-
-
-
+    <string name="dates">Dates</string>
+    <string name="favoured">Favoured</string>
```

Comment:
I was thinking maybe this should be "Only show favourites" or something like that
Maybe wait for other opinions but I feel like Favoured is a odd word to use to relate with Favourites?